code/modules/mob/mob_defines.dm

```diff
@@ -58,7 +58,7 @@
 	var/drowsyness = 0//Carbon
 	var/dizziness = 0//Carbon
 	var/jitteriness = 0//Carbon
-	var/nutrition = NUTRITION_LEVEL_FED + 50//Carbon
+	var/nutrition = rand(NUTRITION_LEVEL_START_MIN, NUTRITION_LEVEL_START_MAX)
```

Comment:
This shouldn't be in defines, it requires constants. not variable things.

Reply:
The definition aspect is in the creation of the variable, not its value.